cloud/google/cmd/gce-machine-controller/README.md

```diff
@@ -0,0 +1,28 @@
+# GCE Machine Controller
+
+The GCE Machine Controller is a machine controller implementation for clusters running on Google Compute Engine.
+
+## Development
+The instructions below will enable you to get started with running your own version of the gce-machine-controller on a GCE Kubernetes cluster.
+
+### Prerequisites
+
+1. Follow the "Before you begin" section from the google cloud container registry [pushing and pulling images documentation](https://cloud.google.com/container-registry/docs/pushing-and-pulling).
+1. From this directory, run `make dev_push`.
```

Comment:
Before running this, a pre-req is gcloud installed, auth'd and default project set up with gcr enabled.

Reply:
Hello @karan, I think those items are covered in the instructions from the google cloud container registry that are linked. I'd prefer to not duplicate as they may change a bit over time.